### PR TITLE
updated server_benchmark to new schemas

### PR DESCRIPTION
**Pull request by bwalsh**
Continuation of discussion on https://github.com/ga4gh/server/pull/755

This PR addresses use of global in server_benchmark.py and leverages utils.captureOutput

### CODE REVIEW

File: tests/unit/test_server_benchmark.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+from __future__ import unicode_literals
+
+import re
+import time
+import pstats
+import argparse
+import cProfile
+import sys
+
+import ga4gh.backend
```
Comment (jeromekelleher) [inline tests/unit/test_server_benchmark.py]: We usually use ``import ga4gh.backend as backend``.

**Comment by jeromekelleher** [comment]
Looks like the same file has been copied in twice here.

**Comment by bwalsh** [comment]
@jeromekelleher  thanks.  I'll address the import statement.  However, I couldn't follow your comment `Looks like the same file has been copied in twice here.` There is one commit with two files, server_benchmark.py and test_server_benchmark.py.  What did I miss?

**Comment by jeromekelleher** [comment]
That was a bit cryptic all right, sorry @bwalsh. It looks like ``tests/unit/test_server_benchmark.py`` and ``scripts/server_benchmark.py`` have the same contents now.

**Comment by bwalsh** [comment]
thanks, that slipped through.  all should be well now

File: server_benchmark.py [hunk 2]
```diff
@@ -113,12 +114,18 @@ def benchmarkOneQuery(request, repeatLimit=3, pageLimit=3):
     # return sum(times[2:])/len(times[2:])
     return min(times)
 
-if __name__ == '__main__':
+
+def main(args):
```
Comment (jeromekelleher) [inline server_benchmark.py]: It's a bit confusing if we have ``args`` referring to two different things here. Maybe change this parameter to ``argList`` or something?

**Comment by jeromekelleher** [comment]
Looks great, thanks @bwalsh. +1 modulo the minor nitpick above.

**Comment by jeromekelleher** [comment]
@bwalsh, can we update this please? It would be good to merge this, and it just needs a tiny change and a rebase.

**Comment by bwalsh** [comment]
Yes.  I've been slammed on other items, but just getting my head above
water now.

-b

-Brian Walsh

**Comment by jeromekelleher** [comment]
Closing this PR due to inactivity, and because we need to move our default branch to ``master`` (#771 and #834).

@bwalsh, if you would still like to finish this up, could you reopen this PR against ``master`` please?

**Comment by kozbo** [comment]
@bwalsh I want to play around with this benchmarking framework. Were your changes ever merged? If not then where can I find them?

**Pull request closed without merging**